src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -28,71 +29,184 @@ export default View.extend({
+              {{i18n code="oie.enroll.okta_verify.setup.skipAuth.openOv.suchAs"
+              bundle="login"
+              arguments="enrolledDeviceName"
+              $1="<span class='semi-strong'>$1</span>"}}
+            </li>
+            <li>{{i18n code="oie.enroll.okta_verify.setup.skipAuth.selectAccount" bundle="login"}}</li>
+            <li>
+              {{i18n code="oie.enroll.okta_verify.setup.skipAuth.addAccount"
+              bundle="login" $1="<span class='semi-strong'>$1</span>"}}
+            </li>
+            <li>{{i18n code="oie.enroll.okta_verify.setup.skipAuth.followInstruction" bundle="login"}}</li>
+          </ol>
+          <p class="closing">
+            {{i18n code="oie.enroll.okta_verify.setup.skipAuth.canBeClosed" bundle="login"}}
+          </p>
+        {{/if}}
       {{/if}}
     `,
+  /* eslint complexity: [2, 20] */
   getTemplateData() {
     const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
     let enrolledDeviceName = '';
-    if (contextualData && contextualData?.devicebootstrap && contextualData?.devicebootstrap.enrolledDevices) {
-      const enrolledDevices = contextualData?.devicebootstrap.enrolledDevices;
-      enrolledDeviceName = Array.isArray(enrolledDevices) && !_.isEmpty(enrolledDevices) ?
-        enrolledDevices[0] : enrolledDevices;
+    let deviceMap = [];
+    let showAnotherDeviceLink = false;
+    if (contextualData) {
+      if (contextualData?.devicebootstrap && contextualData?.devicebootstrap.enrolledDevices) {
+        const enrolledDevices = contextualData?.devicebootstrap.enrolledDevices;
+        enrolledDeviceName = Array.isArray(enrolledDevices) && !_.isEmpty(enrolledDevices) ?
+          enrolledDevices[0] : enrolledDevices;
+      }
+      if (contextualData.samedevice && contextualData.samedevice?.setupOVUrl) {
+        deviceMap = contextualData.samedevice;
+      } else if (contextualData.devicebootstrap && contextualData.devicebootstrap?.setupOVUrl) {
+        deviceMap = contextualData.devicebootstrap;
+      }
+    }
+
+    if (deviceMap.platform) {
+      deviceMap.platformLC = deviceMap.platform.toLowerCase();
+      deviceMap.isDesktop = !(deviceMap.platformLC === 'ios' || deviceMap.platformLC === 'android');
     }
 
+    if (deviceMap.securityLevel && deviceMap.securityLevel === 'ANY') {
+      showAnotherDeviceLink = true;
+    }
     return {
       href: contextualData.qrcode?.href,
       email: _.escape(contextualData?.email),
       phoneNumber:  _.escape(contextualData?.phoneNumber),
       sameDevice: contextualData?.samedevice,
       deviceBootstrap: contextualData?.devicebootstrap,
       enrolledDeviceName: enrolledDeviceName,
+      deviceMap: deviceMap,
+      showAnotherDeviceLink: showAnotherDeviceLink,
     };
   },
   postRender: function() {
+    const appState = this.options && this.options.appState;
+    const model = this.model;
+
+    $(document).on('click', '.orOnMobileLink', function() {
```

Comment:
Is there a reason we have to attach the click event listener to the `document` or can it be attached closer to the `.orOnMobileLink` class?

Reply:
this is the only option that worked

Reply:
Does this work? 

```
$('.orOnMobileLink').click(function() {
```

Reply:
Or alternatively (using arrow function if there are variables that need to be bound to the callback); 

```
$('.orOnMobileLink').click(() => {
```